src/components/shared/renderers/Checkbox.jsx

```diff
@@ -0,0 +1,29 @@
+//@flow
+import React from 'react';
+import './checkbox-styles.scss';
+
+type Props = {
+  checked: boolean,
+  children: any,
+  onChange: (key: string, checked: boolean) => void,
+};
+
+const Checkbox = ({checked, children, onChange}: Props) => {
+  const key = children[0].props.href;
+  const toggleCheckbox = () => onChange(key, !checked);
+
+  return (
+    <li className="checkbox">
+      <label className="checkbox__wrapper" onClick={toggleCheckbox}>
+        <div
+          className={`checkbox__toggle ${
+            checked ? 'checkbox__toggle--checked' : ''
+          }`}
+        />
+        <div className="checkbox__label">{children.slice(1)}</div>
+      </label>
```

Comment:
I think we might want this label to only cover the `checkbox__toggle`... if there is an indented checklist those actually render inside of `{children.slice(1)}` and with the `<label>` wrapping those, any clicks in them bubble up to the parent.

Either that, or we need to get more fancy with breaking down the `children` but I think moving the checkbox to be just the checkbox and not the row content should suffice for us.

Reply:
Good point. I think that would make sense to only check the toggle.